helios-client/src/main/java/com/spotify/helios/common/JobValidator.java

```diff
@@ -150,7 +150,7 @@ public JobValidator(final boolean shouldValidateJobHash,
         errors.add("Volume path is not absolute: " + path);
         continue;
       }
-      if (!isNullOrEmpty(source) && !source.startsWith("/")) {
+      if (isNullOrEmpty(source) && !source.startsWith("/")) {
```

Comment:
@rohansingh Ah, yes, you're right! In our use case with the named volume I've removed the second condition completely and there it makes sense!